content/en/docs/ui-reference-guide/GELI.md

```diff
@@ -0,0 +1,173 @@
+---
+Title: "GELI Encryption in FreeNAS/TrueNAS"
+linkTitle: "GELI Encryption"
+type: docs
+---
+
+{{% alert title="Notice" color="info" %}}
+TrueNAS replaced GELI encryption with ZFS native cryptography in the version 12.0 release.
+This article is provided as a historical reference for encryption management in FreeNAS/TrueNAS 11.3 and earlier.
+{{% /alert %}}
+
+## GELI Encryption
+
+FreeNAS/TrueNAS 11.3 and earlier used
+`GELI <https://www.freebsd.org/cgi/man.cgi?query=geli>`__ to provide
+full disk encryption for ZFS pools. This type of encryption is
+intended to protect against the risks of data being read or copied when
+the system is powered down, when the pool is locked, or when disks are
+physically stolen.
+
+GELI encrypts disks and pools, not individual filesystems. The partition
+table on each disk is not encrypted, but only identifies the location of
+partitions on the disk. On an encrypted pool, the data in each partition
+is encrypted. These are generally called "encrypted drives", even though
+the partition table is not encrypted.
+
+Using a processor that supports the
+`AES-NI <https://en.wikipedia.org/wiki/AES_instruction_set>`__
+instruction set is strongly recommended. These processors can handle
+encryption of a small number of disks with negligible performance impact.
+They also retain performance better as the number of disks increases.
+Older processors without the AES-NI instructions see significant
+performance impact with even a single encrypted disk. This
+`forum post <https://forums.freenas.org/index.php?threads/encryption-performance-benchmarks.12157/>`__
+compares the performance of various processors.
+
+All drives in a GELI encrypted pool are encrypted, including L2ARC (read
+cache) and SLOG (write cache). Drives added to an existing encrypted
+pool are encrypted with the same method specified when the pool was
+created. Data in memory, including ARC, is not encrypted. ZFS data on
+disk, including L2ARC and SLOG, are encrypted if the underlying disks
+are encrypted. Swap data on disk is always encrypted.
+
+Encryption performance depends upon the number of disks encrypted. The
+more drives in an encrypted pool, the more encryption and decryption
+overhead, and the greater the impact on performance. **Encrypted pools
+composed of more than eight drives can suffer severe performance
```

Comment:
Verify this, I believe this was only true of the older GELI encryption method.  I don't know if the newer one has the same issue.

Reply:
This is all true related to GELI except for the last sentence that says `encrypted pools composed of more than eight drives can suffer severe performance penalties` We have multiple truenas customers that are using 50+ disk in GELI encrypted zpools with no perceived performance issues. I would suggest that we leave any specific HDD numbers off the documentation for 2 reasons.
1. our customers will read this and think they have a problem and say "well, I've got 75 disks in my pool" so I must have a problem.
2. the actual number of HDD's before GELI encryption penalties are seen is, quite literally, never been quantified in any scientific manner so it's all just hear-say.

As a matter of fact, the performance team did tests with a 100 disk GELI pool and the performance numbers were almost as fast as a pool with no GELI encryption. I'm being long-winded to say, I'd recommend you remove the 8 drive number as that will only cause confusion.

Reply:
Can do!